util/chplenv/chpl_libfabric.py

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env python
 import sys
+from sys import stderr
```

Comment:
Nit, but personally it doesn't seem worthwhile to import stderr to me. I'd probably just write below as `sys.stderr.write`